src/components/Modals/FundsModal/FundsModal.scss

```diff
@@ -0,0 +1,32 @@
+.funds-modal {
+    @include default-modal;
+
+    &__header {
+        padding-top: 1rem !important;
+
+        @include mobile-or-tablet-screen {
+            padding-top: 0 !important;
+            padding-left: 1.6rem;
```

Comment:
need to use important here, for some reason without it, the header stylings couldn't be overriden (for the padding-top)

Reply:
i think the below should work.
```
.deriv-modal__header {
        padding-top: 0;
    }
```